src/server/input_service.rs

```diff
@@ -1571,6 +1571,13 @@ async fn send_sas() -> ResultType<()> {
     Ok(())
 }
 
+#[tokio::main(flavor = "current_thread")]
+pub async fn connect_to_user_session(usid:Option<u32>) -> ResultType<()> {
```

Comment:
should put connect_to_user_session here. This is not part of input_service. Put in ipc.rs.